README.md

```diff
@@ -10,129 +21,44 @@ Make sure you have TypeScript and @typescript-eslint/parser installed, then inst
 npm i -D eslint-plugin-expect-type
 ```
 
-Please also make sure the following packages are also installed:
-
-```sh
-npm i -D eslint @typescript-eslint/eslint-plugin @typescript-eslint/parser
-```
+> See [typescript-eslint's Getting Started docs](https://typescript-eslint.io/docs) for how to run ESLint on TypeScript files.
 
 ## Usage
 
-Please add the following options to your `.eslintrc`
-
-```json
-{
-  "extends": ["plugin:eslint-plugin-expect-type/recommended"],
-  "parser": "@typescript-eslint/parser",
-  "parserOptions": {
-    "project": "./tsconfig.json"
-  },
-  "plugins": ["eslint-plugin-expect-type"],
-  "root": true
-}
-```
-
-Rule severity could be configured as follows
+Add the following options to your [ESLint configuration file](https://eslint.org/docs/latest/user-guide/configuring/configuration-files):
 
 ```json
 {
-  "rules": {
-    "expect-type/expect": "error"
-  }
+  "extends": ["plugin:expect-type/recommended"],
+  "plugins": ["eslint-plugin-expect-type"]
 }
 ```
 
-To skip Snapshot update when eslint is run with `--fix` could be configured as follows:
-
-```json
-{
-  "rules": {
-    "expect-type/expect": ["error", { "disableExpectTypeSnapshotFix": true }]
-  }
-}
-```
-
-**Note: Make sure to use `eslint --ext .js,.ts` since by [default](https://eslint.org/docs/user-guide/command-line-interface#--ext) `eslint` will only search for .js files.**
-
-## Adding $ExpectType and $ExpectError type assertions
-
-> A test file should be a piece of sample code that tests using the library. Tests are type-checked, but not run. To assert that an expression is of a given type, use $ExpectType. To assert that an expression causes a compile error, use $ExpectError. (Assertions will be checked by the expect lint rule.)
-
-(https://github.com/Microsoft/dtslint#write-tests)
-
-```ts
-import foo from 'lib-to-test'; // foo is (n: number) => void
-
-// $ExpectType void
-foo(1);
-
-// Can also write the assertion on the same line.
-foo(2); // $ExpectType void
-
-// $ExpectError
-foo('bar');
-```
-
-## Adding \$ExpectTypeSnapshot
-
-Uses snapshot saved in file as expected type for expression.
-
-Example:
-
-**foo.test.ts**
-
-```ts
-// $ExpectTypeSnapshot MyFooSnapshot
-const Foo = {
-  a: 1,
-  n: 17,
-} as const;
-```
-
-By running `eslint --fix` the following file will be created in the folder of `foo.test.ts`:
+Then, you'll be able to use `^?`, `$ExpectError`, `$ExpectType`, and `$ExpectTypeSnapshot` comments in code assert on types.
 
-```plaintext
-__type-snapshots__/foo.test.ts.snap.json
-```
-
-By running `eslint` snapshot type will be matched with actual type and Error will be emitted in case types don't match.
+<!-- prettier-ignore-start -->
```

Comment:
In case you wanted to avoid disabling prettier for this, there are a couple ways to solve this in: 

* https://github.com/bmish/eslint-doc-generator#prettier

Example:
* https://github.com/jest-community/eslint-plugin-jest/blob/647a469dc1175f31c00e22b4b15b9c3c6c6bef26/.eslint-doc-generatorrc.js#L21

Reply:
Ah thanks! I'll probably leave this as is for a bit, since it's less config for now. But this is good to know!